centreon/www/front_src/src/Resources/Graph/Performance/AnomalyDetection/AnomalyDetectionEstimatedEnvelopeThreshold.tsx

```diff
@@ -58,10 +58,14 @@ const AnomalyDetectionEstimatedEnvelopeThreshold = ({
     item,
     factors,
   }: EnvelopeVariation): number => {
+    const upperRealValue = prop(metricUpper, item) as number;
+    const lowerRealValue = prop(metricLower, item) as number;
+    const currentMultiplicationFactor = factors.currentFactor;
+    const simulatedMultiplicationFactor = factors.simulatedFactor;
+
     return (
-      ((prop(metricUpper, item) as number) -
-        (prop(metricLower, item) as number)) *
-      (1 - factors.simulatedFactor / factors.currentFactor)
+      ((upperRealValue - lowerRealValue) / 2) *
```

Comment:
I think Bruno wrote the same comment in a previous pr about the same thing. Can you split the formula to get a better understanding about what it is supposed to do :)

Reply:
I applied the remark, and it was the result, and you had approved it